e2e-environment/Dockerfile

```diff
@@ -39,10 +39,10 @@ ARG CONTRACTS_VERSION
 ARG CONTRACTS_PACKAGE_VERSION
 ARG GETH_VERSION
 
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends supervisor python3-virtualenv \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends supervisor python3-virtualenv libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
```

Comment:
Why do we need GTK in that image?

Reply:
This is for Cypress (did I miss this in the commit messages?).
Therefore I blindly followed [these instructions](https://docs.cypress.io/guides/getting-started/installing-cypress.html#Linux) by them.

Reply:
Nah, I just didn't read them... 🙈